README.md

```diff
@@ -16,16 +16,19 @@
 <!-- /TOC -->
 
 # Description
-Using a tool capable of posting (Postman) post a json body to the apps endpoint `/messages` with a json body like:
+Using a tool capable of making POST requests (Postman), send a POST request to the app's endpoint `/callbacks/outbound/messaging` with a json body like:
 ```json
 {
-  "to": "+19994444",
+  "to": "+19199994444",
```

Comment:
is that a Bandwidth number our account owns?

Reply:
I don't believe so, I think its just a random number tbh